cuebot/src/main/java/com/imageworks/spcue/dao/postgres/HistoricalDaoJdbc.java

```diff
@@ -40,4 +46,34 @@ public void transferJob(JobInterface job) {
          */
         getJdbcTemplate().update("DELETE FROM job WHERE pk_job=?", job.getJobId());
     }
+
+    @Override
+    public List<HistoricalJob> getJobHistory(List<String> shows, List<String> users,
+            List<String> shots, List<String> jobNameRegex, List<JobState> states, long startTime,
+            long endTime, int page, int pageSize, int maxResults) {
+        // Historical queries are handled via Elasticsearch when enabled
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<HistoricalFrame> getFrameHistory(String jobId, String jobName,
+            List<String> layerNames, List<FrameState> states, long startTime, long endTime,
+            int page, int pageSize) {
+        // Historical queries are handled via Elasticsearch when enabled
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<HistoricalLayer> getLayerHistory(String jobId, String jobName, long startTime,
+            long endTime, int page, int pageSize) {
+        // Historical queries are handled via Elasticsearch when enabled
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<LayerMemoryRecord> getLayerMemoryHistory(String layerName, List<String> shows,
+            long startTime, long endTime, int maxResults) {
+        // Historical queries are handled via Elasticsearch when enabled
+        return Collections.emptyList();
+    }
```

Comment:
I don't see these function implemented anywhere. When Elasticsearch is enabled, where is this being queried?